docs/paid-features/activating-the-enterprise-edition.md

```diff
@@ -20,7 +20,11 @@ You can get a license by signing up for a free trial of the [Pro or Enterprise e
 
 ## How to activate your token when self-hosting
 
-If you chose to host Metabase yourself, you'll get an email containing a unique license token. But to use it, you'll need to install the right JAR file.
+If you want to host Metabase yourself, you'll get an email containing a unique license token. But to use it, you'll need to install the Enterprise Edition of Metabase.
+
+The upgrade to an Enterprise Edition will keep all of your questions, dashboards, people, settings---everything in your existing Metabase.
```

Comment:
The only thing you need to do is to replace an open source JAR file with an Enterprise Edition JAR, or <what to do if they are on Docker?>, and enter your license token in the Admin Settings -> License section.